scripts/team.js

```diff
@@ -0,0 +1,15 @@
+$(document).ready(function {
+  jQuery.getJSON('https://api.dvcoders.com/kernels', function(data) {
+    var users = data.result;
+  
+    for (i = 0; i < users.length; i++) {
+      var newHtml = '<li class="col-md-4">' +
```

Comment:
Specifically the `var newHtml = ` part

If `var newHtml` was defined outside of the loop and `newHtml +=` was used I think it would not get redefined